cli.js

```diff
@@ -84,7 +84,7 @@ function action(paths, options) {
 
   var filesDone = [];
 
-  for (var j = 0; j < files.length; j++) {
+  for (var i = 0; i < files.length; i++) {
```

Comment:
Why should we use j here as the only place when we use i everywhere else.?
There are 5 for-loops and 4 of them use i for index and this one we changed to j.